baselines/splunk_risky_command_abuse_ml_detection_baseline.yml

```diff
@@ -0,0 +1,71 @@
+name: Splunk Risky Command Abuse ML Detection Baseline
+id: 273df2f7-643a-451a-8d4d-637e39eadc87
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Baseline
+datamodel:
+- Splunk_Audit
+description: 'This search is to build a per user risky command abuse detection model using MLTK DensityFunction 
+  algorithm based on audit log data of Splunk apps. The model is built from past 7 days users history of running 
+  search commands containing risky keywords, and aggregate the total search run time for each hour as indicator of 
```

Comment:
Rename from "search commands containing risky keywords" to "search commands from https://docs.splunk.com/Documentation/Splunk/latest/Security/SPLsafeguards#Commands_that_trigger_the_warninga"

Reply:
done